tests/STA/perfect_sim.py

```diff
@@ -2,11 +2,13 @@
 import logging
 
 from Util import Pose, Position
-from Util.constant import ROBOT_RADIUS, ROBOT_CENTER_TO_KICKER, BALL_RADIUS
+from Util.constant import ROBOT_CENTER_TO_KICKER, BALL_RADIUS
 from Util.geometry import compare_angle
-from ai.GameDomainObjects import Ball
 from ai.STA.Tactic.tactic import Tactic
 from ai.states.game_state import GameState
+from config.config import Config
+
+Config().set_default()
```

Comment:
This is dangerous, it would be preferable if every test set the configuration to default. Or to do it inside the construstor.

Reply:
I don't know what to do :( help me